packages/types/src/config.ts

```diff
@@ -813,13 +876,23 @@ export type AdminConfig = {
     DataRetentionSettings: DataRetentionSettings;
     MessageExportSettings: MessageExportSettings;
     JobSettings: JobSettings;
+    ProductSettings: ProductSettings;
     PluginSettings: PluginSettings;
     DisplaySettings: DisplaySettings;
     GuestAccountsSettings: GuestAccountsSettings;
     ImageProxySettings: ImageProxySettings;
+    CloudSettings: CloudSettings;
     FeatureFlags: FeatureFlags;
+    ImportSettings: ImportSettings;
+    ExportSettings: ExportSettings;
 };
 
+export type ReplicaLagSetting = {
+    DataSource: string;
+    QueryAbsoluteLag: string;
+    QueryTimeLag: string;
+}
+
```

Comment:
Updated types here based on latest version (7.5).

Reply:
Wow, I didn't realize how far behind we were on these. We really need to look at getting these generated automatically at some point